apps/pynq_rpc/start_rpc_server_to_tracker_as_ultra96.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
The folder containing this file is still `pynq_rpc`, will you give it a better name?

Reply:
so the issue is that there is a bit of aliasing of the name pynq. It refers to the SoC focused Xilinx linux driver stack (http://www.pynq.io/) which is compatible with a bunch of board including the pynqz1, the ultra96, the zcu104 etc.

One option is to separate the libraries from the board by explicitly naming the target `pynqz1`; the other is to rename the commonly usable RPC code to `zynq`, or `vta_soc` but that might capture design points that are incompatible with the pynq.io libraries

Reply:
what if we have `vta_rpc` then having a specific filename for each bash script?

Reply:
> what if we have vta_rpc then having a specific filename for each bash script?

I think this is a better option.

Reply:
Sounds good, I'll go ahead and rename it